src/ol_infrastructure/applications/xpro/Pulumi.applications.xpro.Production.yaml

```diff
@@ -2,6 +2,55 @@
 secretsprovider: awskms://alias/infrastructure-secrets-production
 encryptedkey: AQICAHiiGjYUolrtj8PCnScLM7oLAdMl8nJrLjQjnqyl1LykYgFDjLJKJAYAWev29dCBcDyuAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMieA4m88ozKuQi3iTAgEQgDuiRRn7fmtTU/I9yXx3ere67Ee32cTL8gbh3JraHuColZVV3FPJZl+ScIjcV7egqP4I+0RidtZhM0t2VA==
 config:
+  heroku:app_id:
+    secure: v1:sflDJ1/1qixNMWt8:zni8/Aoc3kXrp6nBAU2oTyCcCcHrQl/yLYswzg49KlQlrg07oB4FoV/4Fr6PsCcTZ39ttw==
+  heroku:user: "mitx-devops"
+  heroku_app:vars:
+    CSRF_TRUSTED_ORIGINS: "xpro.mit.edu,xpro-web.odl.mit.edu"
+    CYBERSOURCE_SECURE_ACCEPTANCE_URL: "https://secureacceptance.cybersource.com/pay"
+    CYBERSOURCE_WSDL_URL: "https://ics2wsa.ic3.com/commerce/1.x/transactionProcessor/CyberSourceTransaction_1.154.wsdl"
+    DIGITAL_CREDENTIALS_DEEP_LINK_URL: "dccrequest://request"
+    DRIVE_WEBHOOK_EXPIRATION_MINUTES: 1440
+    DRIVE_WEBHOOK_RENEWAL_PERIOD_MINUTES: 720
+    ENABLE_ORDER_RECEIPTS: "True"
+    FEATURE_COUPON_SHEETS_ALT_PROCESSING: "False"
```

Comment:
Not seeing this in pillar

Reply:
I don't think `FEATURE_COUPON_SHEETS_ALT_PROCESSING` is a functioning feature flag. Perhaps @arslanashraf7 or @asadali145 could confirm.

Reply:
@pdpinch `FEATURE_COUPON_SHEETS_ALT_PROCESSING` isn't being used.

Reply:
> Not seeing this in pillar

Probably came out of heroku then during reconciliation.